features/configuration_files/schema_validation.feature

```diff
@@ -5,7 +5,7 @@ Feature: Validate schema
 
   Scenario: Our generated default configuration
     Given our default configuration file
-    And the clean file 'clean.rb'
+    And the clean file "clean.rb"
```

Comment:
Boyscout rule here and below, you should be able it whatever you like and double quotes are cucumber convention.